### DISCUSSION TITLE
Windows Domains Password Management

### DISCUSSION THREAD

**Discussion by HassanMullah** [dani-garcia/vaultwarden#1292] [Ideas]
In our company, we have to change the Windows domain password about every 90 days. This password is required both directly in Windows and on various intranet sites. This means that I have several entries where the same password is always stored for 90 days.

In KeePass 2, it is possible to reference a field of a master entry, i.e. I only enter the new password in the master entry and all other entries look in the master entry.

Does Bitwarden also have something like this, or how do you usually solve this?

**Comment by nulano** [comment]
If it's the same username and password, you can just add multiple URIs for a single entry.

**Reply by HassanMullah** [reply]
OK, I roughly understood, i.e. I create an entry for e.g. URL 1 and then add URL 2 to ..... 14, 15, 16 etc.

This is all OK as long as we are only using URLs in the browser, but in Windows we also have network access etc. where we also need the user name and password. What support does Bitwarden offer here?

**Reply by nulano** [reply]
All services that Bitwarden can auto-fill have a URI associated with them, that is entered into these fields, such as `androidapp://com.spotify.music`. However, while I would love to be proven wrong, I don't think Bitwarden can auto-fill in Windows (outside a browser).
